arches/app/models/models.py

```diff
@@ -715,6 +715,9 @@ class NodeGroup(models.Model):
         null=True,
         on_delete=models.CASCADE,
     )  # Allows nodegroups within nodegroups
+    graph = models.ForeignKey(
+        GraphModel, db_column="graphid", blank=True, null=True, on_delete=models.CASCADE
```

Comment:
Also, null=True doesn't really gel with on_delete=cascade, so if it's only here to support preexisting cruft data, and we already have a major version bump, maybe we just add a command to manage.py validate that deletes cruft nodegroups, and say you have to run that fix before upgrading to 8? WDYT?